### ISSUE TITLE
chore: Code Improvements

### ISSUE THREAD

**Issue by pranshuchittora** [react-native-elements/playground#3]
### Description
The project is in its initial phase, and the code is written with an idea of `move fast and break things`. So there are many possible improvements. Mentioning some but not limited to these only :smile: 

- Inline styling to the stylesheet
- Proper Scoping and Abstraction of composable components.
- Improve folder structure, if any
- Improve UI/UX

**Comment by saideepesh000** [comment]
Hey @pranshuchittora, I would like to work on this, just wanted to know, which should I work among the options. Just assign me the problem so that I can work accordingly

**Comment by pranshuchittora** [comment]
Thanks @saideepesh000 you can start by migrating the inline styling to style objects

**Comment by saideepesh000** [comment]
@pranshuchittora, it would be better if we add CSS modules, it looks clean and beginner-friendly, what do you think.

**Comment by pranshuchittora** [comment]
Ya sounds great :+1: 
Go ahead and feel free to ping me on slack for any help/discussion :)

**Comment by saideepesh000** [comment]
Hi @pranshuchittora, I am getting this error. It seems to the problem related to mdx, how can I tweak the node_modules or replace mdx.  Sorry for this silly doubt. I am using windows(this might also be the problem). And according to [this ](https://github.com/frontarm/mdx-util/issues/44)even though we are using react-app-rewired, it's not working for me. Please help me.

![Screenshot (283)](https://user-images.githubusercontent.com/43727167/99555162-02230b00-29e6-11eb-81d0-c3488b72525e.png)

**Comment by pranshuchittora** [comment]
No worries, let's connect on Slack to discuss and fix the issue.

https://react-native-elements-slack.herokuapp.com/

**Comment by ARJUN-SABU** [comment]
Hi @pranshuchittora, I have gone through the issues mentioned above and i feel that i can start working on them.😋

**Comment by shivambalwani** [comment]
hey @pranshuchittora, I have started working on the issues mentioned above and will be contributing soon.😊

**Comment by pranshuchittora** [comment]
Glad to hear that.
It would be great if you
- Open a draft PR while you are working
- Make small PRs for fast review & merger

**Comment by Devesh21700Kumar** [comment]
I’ve started working on them and have made a PR for certain areas.. will fix the inline styling and stuff in the next PR

**Comment by abhishekkumar08** [comment]
> 
> 
> Hi @pranshuchittora, I am getting this error. It seems to the problem related to mdx, how can I tweak the node_modules or replace mdx. Sorry for this silly doubt. I am using windows(this might also be the problem). And according to [this ](https://github.com/frontarm/mdx-util/issues/44)even though we are using react-app-rewired, it's not working for me. Please help me.
> 
> ![Screenshot (283)](https://user-images.githubusercontent.com/43727167/99555162-02230b00-29e6-11eb-81d0-c3488b72525e.png)

Hey @pranshuchittora @saideepesh000 I'm facing the exact issue, did you found the solution? I search the slack chats but couldn't find it. If any solution please tell me.

**Comment by Devesh21700Kumar** [comment]
> > Hi @pranshuchittora, I am getting this error. It seems to the problem related to mdx, how can I tweak the node_modules or replace mdx. Sorry for this silly doubt. I am using windows(this might also be the problem). And according to [this ](https://github.com/frontarm/mdx-util/issues/44)even though we are using react-app-rewired, it's not working for me. Please help me.
> > ![Screenshot (283)](https://user-images.githubusercontent.com/43727167/99555162-02230b00-29e6-11eb-81d0-c3488b72525e.png)
> 
> Hey @pranshuchittora @saideepesh000 I'm facing the exact issue, did you found the solution? I search the slack chats but couldn't find it. If any solution please tell me.

import {importMDX} from "mdx.macro"; 
use it to work locally

while pushing convert it back to import importMDX from "mdx.macro"; to avoid build errors
@pranshuchittora  this is what i did.. and i think its the best solution right now unless tsc migration is made

**Comment by abhishekkumar08** [comment]
@Devesh21700Kumar yeah it did worked 🚀 , but had to change three instances of it to run.
```
 src/content/Avatar/index.jsx
 src/content/Badge/index.jsx 
 src/content/Button/index.jsx
```

**Comment by Devesh21700Kumar** [comment]
Great.. yes i know. and before pushing revetr it back to import {importMDX} or it will give build error

@pranshuchittora the build error in my PR yesterday was due to this only.. so tweaked a bit to find a solution 🚀

**Comment by khushal87** [comment]
@Devesh21700Kumar I did the same, is there some issue with the rating component? Like it was working for all others but it is not able to fetch the cached rating.md.

**Comment by jugshaurya** [comment]
> > Hi @pranshuchittora, I am getting this error. It seems to the problem related to mdx, how can I tweak the node_modules or replace mdx. Sorry for this silly doubt. I am using windows(this might also be the problem). And according to [this ](https://github.com/frontarm/mdx-util/issues/44)even though we are using react-app-rewired, it's not working for me. Please help me.
> > ![Screenshot (283)](https://user-images.githubusercontent.com/43727167/99555162-02230b00-29e6-11eb-81d0-c3488b72525e.png)
> 
> Hey @pranshuchittora @saideepesh000 I'm facing the exact issue, did you found the solution? I search the slack chats but couldn't find it. If any solution please tell me.

you should 
- delete node_modules and package.lock.json file
- then run npm i twice(yes twice)
- then delete node_modules once
- then npm i
- then npm start
It should fix this error!!.

or 
wait for it https://github.com/react-native-elements/playground/pull/97 to be merged.
Yes, I know it is weird,

**Comment by vikasazad** [comment]
Hi @pranshuchittora  I also wanted to be help in this can i?
